frontend/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-container fluid>
+    <content-card
+      class="contentCard"
+      :title="$tc('views.pageNotFound.title')"
+      max-width="1200"
+      toolbar
+    >
+      <template #title-actions>
+        <v-btn
+          class="d-md-none"
+          icon
+          :to="{ name: 'profile', query: { isDetail: true } }"
+        >
+          <user-avatar :user="user" :size="36" />
+        </v-btn>
+      </template>
+
+      <v-row>
+        <v-col
+          offset="0"
+          offset-sm="1"
+          offset-md="2"
+          offset-lg="2"
+          offset-xl="2"
+          cols="12"
+          sm="10"
+          md="8"
+          lg="8"
+          xl="8"
+        >
+          <four-zero-four-display />
+        </v-col>
+        <v-col offset-sm="0" sm="12" offset="2" cols="8">
+          <h1 class="font-weight-bold text-center px-8">
```

Comment:
```suggestion
          <p class="font-weight-bold text-center">
```
I think a paragraph is enough (for a headline the content length is to much)